js/ui/pivot_grid/sortable/const.js

```diff
@@ -0,0 +1,15 @@
+export const SortableConst = {
+    IEFieldWidthCorrection: 1,
+    targets: {
+        drag: 'drag',
+    },
+    classes: {
+        areaBox: 'dx-area-box',
+        areaField: 'dx-area-field',
+        fieldsContainer: 'dx-pivotgrid-fields-container',
+        widget: 'dx-widget',
+    },
+    attrs: {
+        treeListItem: 'tree-list-item'
```

Comment:
What does it mean `treeListItem` in this context?

Reply:
Renamed to ```treeViewItem```